Update collection ids to be accepted collections, inside the centrapa… (#749)
centrapay/centrapay-docs: legacy/src/api/payment-requests/payment-requests.md

{% h4 Optional Fields %}

|     Field            |  Type  |                                         Description                                                                 |
| -------------------- | ------ | ------------------------------------------------------------------------------------------------------------------- |
| bitcoinAddress       | String | ★  Address to send Bitcoin, when the "assetType" is `bitcoin.*`                                                     |
| cennzAddress         | String | ★  Address to send CPay, when the "assetType" is `cennznet.*`                                                       |
| wavesAddress         | String | ★  Waves address to send Zap tokens, when the "assetType" is `zap.*`                                                |
| productCodes         | Array  | Supported product codes for the payment request, when the "assetType" is `epay.nzd.*`                               |

> | acceptedCollections  | Array  | [Accepted Collections](###accepted-collection) for the payment request, when the "assetType" is `centrapay.token.*`|

Reviewer: This link does not work when I run the site locally

Author: This has been fixed :)

★  For payment options which specify an address, there's a requirement to make a transaction on an external ledger.
Once you have made that payment, you can use the transaction id to [Pay a Payment Request](#pay) using the legacy payment API.

<a name="payment-condition">

### Accepted Collection

> Payment Requests that have `centrapay.token.*` as a [Payment Option](###payment-option) will also return an array of Accepted Collections. An Accepted Collection can be used to determine if a [Centrapay Token][] is accepted for a particular Payment Request, and what line items the token can be used to purchase.

Reviewer: Will Accepted Collections be returned on the Payment Option or the Payment Request? The first sentence reads as if the Accepted Collections is returned on the Payment Request.

Author: It will be returned on the Payment Option, I will update it to be more clear. :)

Author: I have updated this, take a look and let me know what you think. :)

> Payment Requests that have `centrapay.token.*` as a [Payment Option](###payment-option) will also return an array of Accepted Collections. An Accepted Collection can be used to determine if a [Centrapay Token][] is accepted for a particular Payment Request, and what line items the token can be used to purchase.

Reviewer: ```suggestion
Payment Requests that have `centrapay.token.*` as a [Payment Option](###payment-option) will also return an array of Accepted Collections. An Accepted Collection can be used to determine if a [Centrapay Token][] can be used to pay a Payment Request, and the line items able to be purchased using the token.
```

> If a Payment Request contains a `centrapay.token.*` [Payment Option](###payment-option) an array of Accepted Collections will be present inside the `centrapay.token` Payment Option. The Accepted Collections returned can be used to determine if a [Centrapay Token][] can be used to pay a Payment Request, and the line items able to be purchased using the token.

Reviewer 2: is [Centrapay Token][] correct? are you meaning to link it somewhere?
